src/main/java/com/tterrag/k9/commands/CommandMappings.java

```diff
@@ -89,7 +91,19 @@ protected CommandMappings(String prefix, CommandMappings<M> parent, MappingType
         this.color = parent.color;
         this.downloader = parent.downloader;
     }
-    
+
+    public static CommandMappings<?> getMappingsCommand(String channel) {
+        switch (channel.toLowerCase(Locale.ROOT)) {
```

Comment:
I'd say use this as a fallback, I see no reason why "mcp", "moj", etc would not be valid. Also what about yarn?